templates/named.conf.options.erb

```diff
@@ -153,4 +153,24 @@ logging {
         category default { syslog_errors;  };
         category lame-servers { null; };
 };
+<%- else -%>
```

Comment:
Are you sure this should be an `else` ? 

If a user doesn't enable `query_log_enable`, then the log_channel stuff should activate?

I expected this to be additional things *inside* the `logging {` block